packages/next/organization.jsx

```diff
@@ -25,6 +25,7 @@ const organizationToUserFragment = `
     user {
       id
     }
+    hasAllSpecializations
```

Comment:
it's better to owerride the `GET_ORGANIZATION_TO_USER_LINK_BY_ID_QUERY` in condo app to avoid other apps problems

Reply:
ok 
done